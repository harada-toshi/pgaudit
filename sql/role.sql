\! cp sql/role/role.conf /tmp/config.conf
\! echo "pgaudit.config_file = '/tmp/config.conf'" >> ./tmp_check/data/postgresql.conf
\! rm ./tmp_check/tmp_check.log

\! pg_ctl -w restart -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! psql contrib_regression -q -a < sql/role/role.sql
\! rm /tmp/config.conf

\! grep "OBJECT" ./tmp_check/tmp_check.log > results/role.out

