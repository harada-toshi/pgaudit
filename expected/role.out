LOG:  AUDIT: OBJECT,6,1,READ,SELECT,TABLE,public.foo,SELECT * FROM foo;,<not logged>
LOG:  AUDIT: OBJECT,7,1,READ,SELECT,TABLE,public.foo,"SELECT foo.id, bar.data FROM foo, bar WHERE foo.id = bar.id;",<not logged>
LOG:  AUDIT: OBJECT,8,1,READ,SELECT,TABLE,public.foo,"SELECT foo.id, bar.data, baz.data FROM foo, bar, baz WHERE foo.id = bar.id AND foo.id = baz.id;",<not logged>
